runtime/vm/jvminit.c

```diff
@@ -2035,6 +2035,11 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 				 */
 				vm->runtimeFlags |= J9_RUNTIME_XFUTURE;
 			}
+
+			if (FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_ENABLE_PREVIEW, NULL) >= 0) {
```

Comment:
This option should only be recognized on jdk11+

Reply:
Okay, fixed.

Also removed from `checkClassVersion`: `(classfile->majorVersion >= 55)`.  

If the Java version is less than 11, we will fail either `(classfile->majorVersion > max_allowed_version)` because the max_allowed_version is < 55, or `J9_ARE_NO_BITS_SET(flags, BCT_EnablePreview) || (classfile->majorVersion != max_allowed_version)` because the major version is not the runtime version (i.e. illegally less than 55) or `--enable-preview` is disabled.  cfdump can dump preview files with bogus major versions but I think that's okay.